src/scripts/components/pages/authors.js

```diff
@@ -1,34 +1,40 @@
 import clearDom from '../../helpers/clearDom';
 import renderToDOM from '../../helpers/renderToDom';
 
+const emptyAuthors = () => {
+  const domString = '<h1>No Authors</h1>';
+  renderToDOM('#store', domString);
+};
+
 const showAuthors = (array) => {
   clearDom();
 
+  const parsedAuthorArray = Object.values(array);
```

Comment:
This should already be an array when we get here. I'd trace back to where this comes from, and do `Object.values` there.